builder/job.go

```diff
@@ -50,7 +50,7 @@ var validCommitCommands = map[string]bool{
 type Config struct {
 	DockerfileName string
 	RemoteURL      string
-	RepoName       string
+	RepoName       interface{}
```

Comment:
Cool! I think we can abstract the parser in runconfig to "opts" package so that we can reuse the code here.